SETUP/lint_charsuites.php

```diff
@@ -1,9 +1,12 @@
 #!/usr/bin/env php
 <?php
 $relPath="../pinc/";
+include_once($relPath."base.inc");
 include_once($relPath."CharSuites.inc");
 include_once($relPath."unicode.inc");
 
+use voku\helper\UTF8;
```

Comment:
if you wanted, you could do ```use voku\helper\UTF8 as utf8;``` to avoid updating the rest of the file, but I think I prefer this.

Reply:
I don't think so - `UTF8::` is a specifier to access a static class function whereas `utf8_chr_to_hex()` is a function name.